levelkdev/swapr-dapp
# [Feature] New ecoRouter list

New selection list and router components on ecoRouter

<img width="451" alt="Screenshot 2022-04-01 at 16 37 57" src="https://user-images.githubusercontent.com/23079677/161296339-ec166a09-9b41-4ae5-a79c-bddec9971cec.png">

# To Test

1. Open the page `swap`
2. Choose a trading pair
- [ ] Verify if new selection list appears
- [ ] Verify there is a default option selected
- [ ] Check if there is a Route display (if trading pair has multiple routes)
3. Select a new option
- [ ] Selected route should change
